src/client/Microsoft.Identity.Client/AppConfig/AssertionRequestOptions.cs

```diff
@@ -30,5 +31,11 @@ public class AssertionRequestOptions {
         /// Claims to be included in the client assertion
         /// </summary>
         public string Claims { get; set; }
+
+        /// <summary>
+        /// Capabilities that the client application declares in the assertion.
```

Comment:
```suggestion
        /// Capabilities that the client application has declared. If the callback implementer calls the token issuer using another client application object (e.g. ManagedIdentityApplication or ConfidentialClientApplication), the same capabilities should be used there.
```

Reply:
i merged this and then did a rebase which removed this merge. sorry.